core/src/main/java/org/apache/iceberg/encryption/Ciphers.java

```diff
@@ -27,10 +30,19 @@
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 
 public class Ciphers {
-  private static final int NONCE_LENGTH = 12;
-  private static final int GCM_TAG_LENGTH = 16;
+  public static final int NONCE_LENGTH = 12;
+  public static final int GCM_TAG_LENGTH = 16;
+  public static final String GCM_STREAM_MAGIC_STRING = "AGS1";
+
+  static final byte[] GCM_STREAM_MAGIC_ARRAY =
+      GCM_STREAM_MAGIC_STRING.getBytes(StandardCharsets.UTF_8);
+  static final int GCM_STREAM_HEADER_LENGTH =
+      GCM_STREAM_MAGIC_ARRAY.length + 4; // magic_len + block_size_len
```

Comment:
I thought we had decided on a static block size?

Reply:
correct. But its length is always 4.